posthog/api/test/test_capture.py

```diff
@@ -191,6 +192,9 @@ def test_capture_event(self, kafka_produce):
             },
             self._to_arguments(kafka_produce),
         )
+        log_context = structlog.contextvars.get_contextvars()
```

Comment:
Using [log capture](https://www.structlog.org/en/stable/testing.html) might be a more canonical way to do this but I think this is fine.